core/src/test/java/com/linecorp/armeria/internal/server/annotation/AnnotatedValueResolverTest.java

```diff
@@ -429,6 +472,43 @@ void redundant2(@Param @Default("defaultValue") String var1) {}
         @Get("/r3/:var1")
         void redundant3(@Param Optional<String> var1) {}
 
+        @Get("/attribute-test")
+        void attributeTest(
+
+                @Attribute(value = "successPrefixOtherValuesOfOtherTypeInt", prefix = AttrDummyService.class)
```

Comment:
Could we reorder all annotation attributes so that `prefix` attribute comes before `value` attribute, so that it matches the parameter order of `AttributeKey.valueOf()`? i.e.
```suggestion
                @Attribute(prefix = AttrDummyService.class, value = "successPrefixOtherValuesOfOtherTypeInt")
```